butter-cms.json

```diff
@@ -28,56 +28,56 @@
     "Search Features [Autosuggestion]": "No",
     "Search Features [Result Highlighting]": "No",
     "Search Features [Geospatial Search]": "No",
-    "Search Features [Search within uploads (pdf,doc)]": "No",
+    "Search Features [Search within uploads (pdf,doc)]": "Yes",
     "Image Manipulation": "Yes",
-    "Image Focalpoint support": "No",
-    "Image Facedetection": "No",
+    "Image Focalpoint support": "Yes",
+    "Image Facedetection": "Yes",
     "Optimized Image Encoding": "Yes",
-    "Asset fingerprinting": "?",
+    "Asset fingerprinting": "No",
     "CDN Support": "Yes",
     "Antivirus Scanning": "No",
     "Custom Binary Handler": "No",
     "System Requirements": "NA",
     "Supported Databases": "NA",
     "Clustering": "NA",
     "Docker Support": "NA",
-    "Backup Feature": "No",
-    "Import/Export": "No",
-    "CLI": "No",
-    "SDK [Java]": "No",
+    "Backup Feature": "Yes",
+    "Import/Export": "Yes",
+    "CLI": "Yes",
+    "SDK [Java]": "Yes",
     "SDK [C#]": "Yes",
     "SDK [PHP]": "Yes",
     "SDK [JavaScript]": "Yes",
     "SDK [React]": "Yes",
     "SDK [AngularJS]": "Yes",
-    "SDK [TypeScript]": "No",
+    "SDK [TypeScript]": "Yes",
     "Web Hooks": "Yes",
     "Eventbus": "No",
-    "Bulk Import": "No",
+    "Bulk Import": "Yes",
     "Client Side Forms": "No",
     "Plugin System": "No",
     "Customizable UI": "No",
     "User Management": "Yes",
     "Role Based Permissions": "Yes",
     "Document Level Permissions": "No",
     "OAuth 2.0 Support": "No",
-    "Auditing": "No",
+    "Auditing": "Yes",
     "API Keys": "Yes",
     "Project Support": "Yes",
     "I18N Localized Content": "Yes",
-    "Content Trees": "No",
+    "Content Trees": "Yes",
     "Tagging": "Yes",
     "Content Relations": "1:1, 1:n, n:n, n:1",
     "Nesting of Fields": "Yes",
     "GDPR Statement": "No",
     "GDPR API": "No",
     "Editing Conflict Detection": "No",
-    "Versioning": "No",
-    "Content Scheduling": "No",
+    "Versioning": "Yes",
+    "Content Scheduling": "Yes",
```

Comment:
https://buttercms.com/features/#fastcontentupdate-content-scheduling
https://buttercms.com/kb/scheduling-blog-posts-and-pages